File: programmers/etc/level2/예상_대진표.py
def solution(n,a,b):
    answer = 0
    while a != b:
        answer += 1
        a, b = (a+1)>>1, (b+1)>>1
    return answer


n,a,b = 8,4,7
print(solution(n,a,b)) # 3

File: Java/programmers/etc/level2/예상대진표.java
package CodeTest.Java.programmers.etc.level2;

public class 예상대진표 {
    public int solution(int n, int a, int b) {
        int answer = 0;
        
        while (a != b) {
            answer++;
            a = (a+1)/2; b = (b+1)/2;
        }
        
        return answer;
    }
}
